*: update TiDB Cloud, Coprocessor, TiFlash docs (#23065)
pingcap/docs: coprocessor-cache.md

Author: @huoyao1125 Would you please review the update in `coprocessor-cache.md`? Thanks!

> [!NOTE]
> The sentence wrapped in `CustomContent` will only show in TiDB Cloud Dedicated and TiDB Self-Managed docs.

### View the Grafana monitoring panel

In Grafana, you can see the **copr-cache** panel in the `distsql` subsystem under the `tidb` namespace. This panel monitors the number of hits, misses, and cache discards of the Coprocessor Cache in the entire cluster.

> **Note:**
>

> > The **copr-cache** panel is unavailable for {{{ .starter }}} and {{{ .essential }}} instances.

Author: @huoyao1125 PTAL. Thanks!

> > The **copr-cache** panel is not available on {{{ .starter }}} and {{{ .essential }}} instances.

Author: ```suggestion
> The **copr-cache** panel is currently unavailable for {{{ .starter }}} and {{{ .essential }}} instances.
```

> > The **copr-cache** panel is currently unavailable for {{{ .starter }}} and {{{ .essential }}} instances.

Author: ```suggestion
> The **copr-cache** panel is unavailable for {{{ .starter }}} and {{{ .essential }}} instances.
```

> > Currently, the **copr-cache** panel is not available on {{{ .starter }}} and {{{ .essential }}} instances.

Author: @huoyao1125 PTAL
